### PR TITLE
tcpstat 0.1 (import from head-only)

### PR DESCRIPTION
**Pull request by bfontaine**
See https://github.com/Homebrew/homebrew-head-only/pull/185

### CODE REVIEW

**Comment by mikemcquaid** [comment]
:+1:

File: Library/Formula/tcpstat.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  homepage "https://github.com/jtt/tcpstat"
+  url "https://github.com/jtt/tcpstat/archive/rel-0-1.tar.gz"
+  version "0.1"
+  sha256 "366a221950759015378775862a7499aaf727a3a9de67b15463b0991c2362fdaf"
+
+  head "https://github.com/jtt/tcpstat.git"
+
+  def install
+    system "make"
+    bin.install "tcpstat"
```
Comment (mikemcquaid) [inline Library/Formula/tcpstat.rb]: The usual `make install` question :grinning:

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/tcpstat.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/tcpstat.rb]: Yep tried it but it doesn’t work (uses a wrong `install` option).

**Pull request closed without merging**